### PR TITLE
easy_graphics 0.1.0

### PR DESCRIPTION
**Pull request by SMerrony**
Created via `alr publish` with `alr 2.0.1`

### CODE REVIEW

File: index/ea/easy_graphics/easy_graphics-0.1.0.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+authors = ["Stephen Merrony"]
+maintainers = ["Stephen Merrony <merrony@gmail.com>"]
+maintainers-logins = ["SMerrony"]
+licenses = "MIT OR Apache-2.0 WITH LLVM-exception"
+website = "https://github.com/SMerrony/easy_graphic"
+tags = ["graphics", "image", "ppm", "pam", "gif", "turtle"]
+
+[[depends-on]]
+gnat = "^13.0"
```
Comment (mosteo) [inline index/ea/easy_graphics/easy_graphics-0.1.0.toml]: Shouldn't this one be `>=13`?

Comment (SMerrony) [reply index/ea/easy_graphics/easy_graphics-0.1.0.toml]: I don't know.  It uses `pragma Ada_2022;` so whatever the minimum is to support that.
Could you advise please?

Comment (SMerrony) [reply index/ea/easy_graphics/easy_graphics-0.1.0.toml]: I've made the change you suggested.  Hope that's right.

Comment (mosteo) [reply index/ea/easy_graphics/easy_graphics-0.1.0.toml]: Yes, that's it, thanks.

**Review by mosteo — APPROVED** [review]

**Pull request merged**